language/documentation/book/src/address.md

```diff
@@ -6,17 +6,75 @@ Although an `address` is a 128 bit integer under the hood, Move addresses are in
 
 You can use runtime address values (values of type `address`) to access resources at that address. You *cannot* access modules at runtime via address values.
 
-## Literals
+## Addresses and Their Syntax
 
-`address` literals are 16-byte hex literals, i.e. `0x<hex encoded value>`. For convenience, leading `0`s are added for literals that are too short.
+Addresses come in two flavors, named or numerical. The syntax for a named address follows the
+same rules for any named identifier in Move. The syntax of a numerical address is not restricted
+to hex-encoded values, and any valid numerical literal can be used as an address value.
```

Comment:
I might give a little grammar breakdown or something here to show the different cases